docs/res/guides/custom_model_template.rst

```diff
@@ -3,35 +3,33 @@
 Advanced : Declaring a custom model
 =============================================
 
-To add a custom model, you will need a **.py** file with the following structure to be placed in the *napari_cellseg3d/models* folder:
+.. warning::
+    **WIP** : Adding new models is still a work in progress and will likely not work simply by adding the model in the plugin.
```

Comment:
The last part of this sentence is a bit confusing. I think you can just write "Adding new models is still a work in progress and will likely lead to errors".

Reply:
Done !